src/components/v2/Table/styles.ts

```diff
@@ -22,9 +22,16 @@ export const useStyles = () => {
         padding: 0;
       }
     `,
-    tableWrapperMobile: css`
+    tableWrapperMobile: ({ clickable }: { clickable: boolean }) => css`
       margin-top: ${theme.spacing(6)};
       padding: ${theme.spacing(4, 0)};
+      ${clickable &&
+      `
+        cursor: pointer;
+        :hover {
+          background-color: ${theme.palette.interactive.hover};
+        }
+      `}
```

Comment:
If it's only relevant to mobile devices, we don't need any hover effect since there's no hover state on mobile (it would only briefly show on press)

Reply:
Got you